exercise4/src/View/Login.php

```diff
@@ -0,0 +1,39 @@
+<?php use src\Controller\LoginController; ?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Login</title>
+    <link rel="stylesheet" href="LoginStyle.css">
+</head>
+<body>
+    <div class="login"><br>

```

Comment:
NIT. Aim to make your codebase readable. This is best practice for all developers.